Source/WebKit/Configurations/WebKitSwift.xcconfig

```diff
@@ -58,3 +61,16 @@ PRIVATE_HEADERS_FOLDER_PATH = $(WEBKITSWIFT_HEADERS_FOLDER_PATH);
 PRIVATE_HEADERS_FOLDER_PATH[sdk=macosx*] = $(PRIVATE_HEADERS_FOLDER_PATH_$(WK_PLATFORM_NAME));
 PRIVATE_HEADERS_FOLDER_PATH_macosx = $(WEBKITSWIFT_HEADERS_FOLDER_PATH);
 PRIVATE_HEADERS_FOLDER_PATH_maccatalyst = $(WK_ALTERNATE_FRAMEWORKS_DIR)/$(WEBKITSWIFT_HEADERS_FOLDER_PATH);
+
+SPI_ONLY_IMPORTS_FLAGS = $(SPI_ONLY_IMPORTS_FLAGS_$(WK_PLATFORM_NAME))
+SPI_ONLY_IMPORTS_FLAGS_macosx = $(EXPERIMENTAL_SPI_ONLY_IMPORTS_FLAGS$(WK_MACOS_1300))
+SPI_ONLY_IMPORTS_FLAGS_MACOS_SINCE_1300 = -Xfrontend -experimental-spi-only-imports
+SPI_ONLY_IMPORTS_FLAGS_iphoneos = -Xfrontend -experimental-spi-only-imports
+SPI_ONLY_IMPORTS_FLAGS_iphonesimulator = -Xfrontend -experimental-spi-only-imports
+SPI_ONLY_IMPORTS_FLAGS_maccatalyst = -Xfrontend -experimental-spi-only-imports
+SPI_ONLY_IMPORTS_FLAGS_appletvos = -Xfrontend -experimental-spi-only-imports
+SPI_ONLY_IMPORTS_FLAGS_appletvsimulator = -Xfrontend -experimental-spi-only-imports
+SPI_ONLY_IMPORTS_FLAGS_watchos = -Xfrontend -experimental-spi-only-imports
+SPI_ONLY_IMPORTS_FLAGS_watchsimulator = -Xfrontend -experimental-spi-only-imports
```

Comment:
I think the way I would express this is with an sdk condition:

```
SPI_ONLY_IMPORTS_FLAGS = -Xfrontend -experimental-spi-only-imports;
SPI_ONLY_IMPORTS_FLAGS[sdk=macosx11*] = ;
SPI_ONLY_IMPORTS_FLAGS[sdk=macosx12*] = ;
```

Reply:
Good call!